plugins/kubernetes/src/utils/clusterLinks/formatters/openshift.test.ts

```diff
@@ -15,21 +15,98 @@
  */
 import { openshiftFormatter } from './openshift';
 
-describe('clusterLinks - OpenShift formatter', () => {
+describe('clusterLinks - Openshift formatter', () => {
```

Comment:
they actually capitalize it `OpenShift`, so we should probably keep it that way here and in the vocab file